azure-pipelines.yml

```diff
@@ -85,7 +85,7 @@ jobs:
     displayName: 'install pre-built openblas'
   - script: python -m pip install --upgrade pip setuptools wheel
     displayName: 'Install tools'
-  - script: python -m pip install cython nose pytz pytest pickle5 vulture docutils sphinx==1.8.5 numpydoc
+  - script: python -m pip install cython nose pytz pytest pickle5 vulture docutils sphinx numpydoc
```

Comment:
I'd prefer to pin this at 2.1. Sphinx breaks something on pretty much every release, so not pinning is too annoying. @mattip was going to try a service that sends an automatic PR when a new release of a dependency is made.

Reply:
sgtm